src/base/solver.jl

```diff
@@ -69,8 +69,8 @@ get_optmodel_bounds(opt_model) = (
 
 """
     set_optmodel_bound!(vidx, opt_model;
-        ub::Maybe{Real} = nothing,
-        lb::Maybe{Real} = nothing,
+        lower::Maybe{Real} = nothing,
+        upper::Maybe{Real} = nothing,
```

Comment:
Should all of these be lower_bound and upper_bound as well? I know it is implied by the bound, but then the user needs to remember two kwargs, and not just the one...

Reply:
here the `bound` is in the name already so I omitted it. Does it make sense?

Same goes for `change_bound[s][!]`.

Reply:
Hmm but that is too broad, because S*v = 0 is also a constraint

Reply:
yap. Naming is hard. :D

Reply:
we'll see after everything gets migrated to new names on how necessary the _bound is after that